src/solvers/elasticity_solver.cpp

```diff
@@ -26,7 +26,7 @@ ElasticitySolver::ElasticitySolver(int order, mfem::ParMesh *pmesh)
   displacement.mesh = pmesh;
   displacement.coll = std::make_shared<mfem::H1_FECollection>(order, pmesh->Dimension(), mfem::Ordering::byVDIM);
   displacement.space =
-      std::make_shared<mfem::ParFiniteElementSpace>(pmesh, displacement.coll.get(), pmesh->Dimension());
+      std::make_shared<mfem::ParFiniteElementSpace>(pmesh.get(), displacement.coll.get(), pmesh->Dimension());
```

Comment:
Are we sure mfem isnt taking ownership of this?

Reply:
Nevermind.. tracked down the header and says explicitly its not owned.

Reply:
Yep, this is a thing MFEM expects the users to handle.